### ISSUE TITLE
Breaking bug in CSS handling in 0.13.3

### ISSUE THREAD

**Issue by apatrida** [nksaraf/vinxi#331]
Breaking bug in Vinxi 0.13.3, dev mode completely breaks on gathering CSS (we have some old sass modules, and as much or more of newer Panda-CSS generating one global css file)  No other changes that Vinxi and it explodes on this stack trace :

```
TypeError: Cannot read properties of undefined (reading 'some')
    at findDeps (...myproject.../node_modules/.pnpm/vinxi@0.3.13_@opentelemetry+api@1.8.0_@types+node@20.12.11_ioredis@5.4.1_lightningcss@1.23.0_sass@1.62.1_terser@5.31.0/node_modules/vinxi/lib/manifest/collect-styles.js:103:44)
    at add (...myproject.../node_modules/.pnpm/vinxi@0.3.13_@opentelemetry+api@1.8.0_@types+node@20.12.11_ioredis@5.4.1_lightningcss@1.23.0_sass@1.62.1_terser@5.31.0/node_modules/vinxi/lib/manifest/collect-styles.js:89:10)
    at add_by_url (...myproject.../node_modules/.pnpm/vinxi@0.3.13_@opentelemetry+api@1.8.0_@types+node@20.12.11_ioredis@5.4.1_lightningcss@1.23.0_sass@1.62.1_terser@5.31.0/node_modules/vinxi/lib/manifest/collect-styles.js:97:10)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async findDeps (...myproject.../node_modules/.pnpm/vinxi@0.3.13_@opentelemetry+api@1.8.0_@types+node@20.12.11_ioredis@5.4.1_lightningcss@1.23.0_sass@1.62.1_terser@5.31.0/node_modules/vinxi/lib/manifest/collect-styles.js:124:4)
    at async add (...myproject.../node_modules/.pnpm/vinxi@0.3.13_@opentelemetry+api@1.8.0_@types+node@20.12.11_ioredis@5.4.1_lightningcss@1.23.0_sass@1.62.1_terser@5.31.0/node_modules/vinxi/lib/manifest/collect-styles.js:89:4)
    at async add_by_url (...myproject.../node_modules/.pnpm/vinxi@0.3.13_@opentelemetry+api@1.8.0_@types+node@20.12.11_ioredis@5.4.1_lightningcss@1.23.0_sass@1.62.1_terser@5.31.0/node_modules/vinxi/lib/manifest/collect-styles.js:97:4)
    at async findDeps (...myproject.../node_modules/.pnpm/vinxi@0.3.13_@opentelemetry+api@1.8.0_@types+node@20.12.11_ioredis@5.4.1_lightningcss@1.23.0_sass@1.62.1_terser@5.31.0/node_modules/vinxi/lib/manifest/collect-styles.js:124:4)
    at async add (...myproject.../node_modules/.pnpm/vinxi@0.3.13_@opentelemetry+api@1.8.0_@types+node@20.12.11_ioredis@5.4.1_lightningcss@1.23.0_sass@1.62.1_terser@5.31.0/node_modules/vinxi/lib/manifest/collect-styles.js:89:4)
    at async add_by_url (...myproject.../node_modules/.pnpm/vinxi@0.3.13_@opentelemetry+api@1.8.0_@types+node@20.12.11_ioredis@5.4.1_lightningcss@1.23.0_sass@1.62.1_terser@5.31.0/node_modules/vinxi/lib/manifest/collect-styles.js:97:4)
    at async findDeps (...myproject.../node_modules/.pnpm/vinxi@0.3.13_@opentelemetry+api@1.8.0_@types+node@20.12.11_ioredis@5.4.1_lightningcss@1.23.0_sass@1.62.1_terser@5.31.0/node_modules/vinxi/lib/manifest/collect-styles.js:124:4)
    at async add (...myproject.../node_modules/.pnpm/vinxi@0.3.13_@opentelemetry+api@1.8.0_@types+node@20.12.11_ioredis@5.4.1_lightningcss@1.23.0_sass@1.62.1_terser@5.31.0/node_modules/vinxi/lib/manifest/collect-styles.js:89:4)
    at async add_by_url (...myproject.../node_modules/.pnpm/vinxi@0.3.13_@opentelemetry+api@1.8.0_@types+node@20.12.11_ioredis@5.4.1_lightningcss@1.23.0_sass@1.62.1_terser@5.31.0/node_modules/vinxi/lib/manifest/collect-styles.js:97:4)
    at async findDeps (...myproject.../node_modules/.pnpm/vinxi@0.3.13_@opentelemetry+api@1.8.0_@types+node@20.12.11_ioredis@5.4.1_lightningcss@1.23.0_sass@1.62.1_terser@5.31.0/node_modules/vinxi/lib/manifest/collect-styles.js:124:4)
    at async findDependencies (...myproject.../node_modules/.pnpm/vinxi@0.3.13_@opentelemetry+api@1.8.0_@types+node@20.12.11_ioredis@5.4.1_lightningcss@1.23.0_sass@1.62.1_terser@5.31.0/node_modules/vinxi/lib/manifest/collect-styles.js:141:5)
    at async findStylesInModuleGraph (...myproject.../node_modules/.pnpm/vinxi@0.3.13_@opentelemetry+api@1.8.0_@types+node@20.12.11_ioredis@5.4.1_lightningcss@1.23.0_sass@1.62.1_terser@5.31.0/node_modules/vinxi/lib/manifest/collect-styles.js:170:23)
    at async viteAssets (...myproject.../node_modules/.pnpm/vinxi@0.3.13_@opentelemetry+api@1.8.0_@types+node@20.12.11_ioredis@5.4.1_lightningcss@1.23.0_sass@1.62.1_terser@5.31.0/node_modules/vinxi/lib/manifest/dev-server-manifest.js:46:7)
    at async Object.assets (...myproject.../node_modules/.pnpm/vinxi@0.3.13_@opentelemetry+api@1.8.0_@types+node@20.12.11_ioredis@5.4.1_lightningcss@1.23.0_sass@1.62.1_terser@5.31.0/node_modules/vinxi/lib/manifest/dev-server-manifest.js:216:16)
    at async createPageEvent (...myproject.../node_modules/.pnpm/@solidjs+start@1.0.2_rollup@4.18.0_solid-js@1.8.18_vinxi@0.3.13_@opentelemetry+api@1.8.0_@typ_j7gkhlis3gkjpjqxi7uxd6scxy/node_modules/@solidjs/start/dist/server/pageEvent.js:33:117)
    at async eval (...myproject..../node_modules/.pnpm/@solidjs+start@1.0.2_rollup@4.18.0_solid-js@1.8.18_vinxi@0.3.13_@opentelemetry+api@1.8.0_@typ_j7gkhlis3gkjpjqxi7uxd6scxy/node_modules/@solidjs/start/dist/server/handler.js:45:25)
    at async _callHandler (...myproject.../node_modules/.pnpm/h3@1.11.1/node_modules/h3/dist/index.mjs:1821:16)
    at async _callHandler (...myproject.../node_modules/.pnpm/h3@1.11.1/node_modules/h3/dist/index.mjs:1821:16)
    at async _callHandler (...myproject.../node_modules/.pnpm/h3@1.11.1/node_modules/h3/dist/index.mjs:1821:16)
    at async ...myproject.../node_modules/.pnpm/h3@1.11.1/node_modules/h3/dist/index.mjs:1962:19
    at async Object.callAsync (...myproject.../node_modules/.pnpm/unctx@2.3.1/node_modules/unctx/dist/index.mjs:72:16)
    at async Server.toNodeHandle (...myproject.../node_modules/.pnpm/h3@1.11.1/node_modules/h3/dist/index.mjs:2249:7)

```

**Comment by apatrida** [comment]
`collect-styles.js` line 103 `sourcesContent` is undefined?

```
if (
		node.url.endsWith(".css") ||
		node.transformResult?.map?.sourcesContent.some((code) => code.match(IGNORE_COMMENT_REGEXP))
	) {
		return;
	}
```

**Comment by apatrida** [comment]
Looks like already being fixed
https://github.com/nksaraf/vinxi/commit/c745546de1e30f8938f165e06364cc900e9fc535

**Comment by ryansolid** [comment]
And released in 0.3.14 https://github.com/nksaraf/vinxi/releases/tag/vinxi%400.3.14
